src/App.js

```diff
@@ -1,9 +1,18 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] I noticed that you disabled **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1: